ppb/sprites.py

```diff
@@ -140,7 +140,34 @@ def _attribute_gate(self, attribute, bad_sides):
             raise AttributeError(message)
 
 
-class BaseSprite(EventMixin):
+class Rotatable:
```

Comment:
`Rotatable` sounds weird to me, but I'm a silly non-native speaker.  *shrugs*

Reply:
Native speaker, sounds weird to me, too, but lots of mixins are named based on their capabilities.

Reply:
-able is approximately the right convention, but it leads to lots of weird words.

I would also accept CanBe*? Also weird.

Reply:
So like `CanBeRotated`?

Reply:
Or just `CanRotate`?  In any case, I don't think `Rotatable` is a problem, I was just very confused whether there was a type, or it was just weird, or I was just bad at English.